### ISSUE TITLE
Small CSS difference

### ISSUE THREAD

**Issue by AntonyXSI** [bwp-codes/oxygen-cli#10]
Hello,

I'm getting a small difference between the cached CSS files created with CLI vs the Oxygen Settings page with the following CSS missing in the CLI created files:

```
div.ct-section-inner-wrap,
div.oxy-header-container {
    max-width: 1400px
}
```

It looks like this CSS might be added by the ct_css_styles function in the component-init.php file in Oxygen?

I'm also receiving some PHP warnings. But do not know if these are causing any issues:

`PHP Warning:  strpos(): Empty needle in oxygen-cli-main/commands/class-oxygenregeneratecsscache.php on line 63
Warning: strpos(): Empty needle in oxygen-cli-main/commands/class-oxygenregeneratecsscache.php on line 63
Warning: Invalid argument supplied for foreach() in oxygen/component-framework/includes/oxygen-dynamic-shortcodes.php on line 430`

**Comment by mickaelperrin** [comment]
Hi @AntonyXSI,

this is indeed a known issue and I finally had some time to work on it.

I just pushed a PR that fix it. see #11

**Comment by bwp-codes** [comment]
Hi @AntonyXSI and @mickaelperrin,
I just merged [PR#11](https://github.com/bwp-codes/oxygen-cli/pull/11) by @mickaelperrin and things should be working again.
@mickaelperrin Thanks again for your contribution! ❤️
